src/Service/PaymentMethodsFilterService.php

```diff
@@ -0,0 +1,96 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Shopware\Service;
+
+use Adyen\Shopware\Handlers\OneClickPaymentMethodHandler;
+use Shopware\Core\Checkout\Payment\PaymentMethodCollection;
+use Shopware\Core\Checkout\Payment\PaymentMethodEntity;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class PaymentMethodsFilterService
+{
+    private PaymentMethodsService $paymentMethodsService;
```

Comment:
Just caught this, typed properties are new in php 7.4, this would break older versions
```suggestion
    private $paymentMethodsService;
```

Reply:
forgot to adjust to the pre PHP 7.4 here 😄 Thanks for catching it. we usually only work with 7.4 mostly now, so I really had to take care to adjust to pre 7.4 code patterns.

Reply:
@peterojo forgot to adjust to the pre PHP 7.4 here 😄 Thanks for catching it. we usually only work with 7.4 mostly now, so I really had to take care to adjust to pre 7.4 code patterns.

should be fixed now.